_data/companies.yml

```
@@ -1509,3 +1509,10 @@
   visitors: Restricted
   events: Restricted
   last_update: 2020-03-12
+
+- name: Welltory
+  wfh: Encouraged
+  travel: Restricted
+  visitors: Restricted
+  events: Restricted
+  last_update: 2020-03-11
```

```suggestion
  last_update: 2020-03-11
```

Hoping that doing this will add the trailing new line properly.